itou/metabase/tables/approvals.py

```diff
@@ -13,61 +13,30 @@
     get_model_field,
     hash_content,
 )
-from itou.prescribers.models import PrescriberOrganization
 
 
 POLE_EMPLOI_APPROVAL_MINIMUM_START_DATE = datetime(2018, 1, 1)
 
 
-@functools.cache
-def get_code_safir_to_pe_org():
-    # Preload association for best performance and to avoid having to make
-    # PoleEmploiApproval.pe_structure_code a foreign key.
-    return {
-        org.code_safir_pole_emploi: org
-        for org in PrescriberOrganization.objects.filter(code_safir_pole_emploi__isnull=False).only(
-            "code_safir_pole_emploi", "department"
-        )
-    }
-
-
 @functools.cache
 def get_companies_map():
     return {c.pk: c for c in Company.objects.only("kind", "siret", "name", "brand", "department")}
 
 
 def get_company_from_approval(approval):
-    if isinstance(approval, PoleEmploiApproval) or not approval.last_hiring_company_pk:
+    if not approval.last_hiring_company_pk:
```

Comment:
La méthode est simplifiable en :
```suggestion
    return get_companies_map().get(approval.last_hiring_company_pk)
```

Reply:
ça ajoute une queriy quand il n'y a que des PASS sans `last_hiring_company_pk` (mais c'est donc fortement improbable et pas grave)